hiera.yaml

```diff
@@ -13,7 +13,9 @@ hierarchy:
   - name: "Terraform data"
     path: "terraform_data.yaml"
   - name: "OS version"
-    path: "os/%{facts.os.name}/%{facts.os.release.major}.yaml"
+    path: "computecanada/os/%{facts.os.name}/%{facts.os.release.major}.yaml"
```

Comment:
@cmd-ntrf This is working right now, the last thing I need is a way to pass through a choice of `computecanada` or `eessi` from terraform to this file